examples/credential_metadata_ldp_vc.json

```diff
@@ -16,35 +16,44 @@
         "type": [
           "VerifiableCredential",
           "UniversityDegreeCredential"
+        ]
+      },
+      "@context": [
```

Comment:
@danielfett 
`@context` is not supposed to be here. its already specified under `credential_definition`
Also please fix the indentation